Description: Create a default etcd.conf to be installed in /etc/etcd
Author: Matt Bostock <matt.bostock@digital.cabinet-office.gov.uk>

Index: etcd-0.4.6/etcd.conf
===================================================================
--- /dev/null 2014-10-23 13:02:00.000000000 +0100
+++ etcd-0.4.6/etcd.conf  2014-10-23 13:00:11.000000000 +0100
@@ -0,0 +1,32 @@
+addr = "127. 0.0.1:4001"
+bind_addr = "127.0.0.1:4001"
+ca_file = ""
+cert_file = ""
+cors = []
+cpu_profile_file = ""
+data_dir = "/var/lib/etcd"
+discovery = ""
+http_read_timeout = 10.0
+http_write_timeout = 10.0
+key_file = ""
+peers = []
+peers_file = ""
+max_cluster_size = 9
+max_result_buffer = 1024
+max_retry_attempts = 3
+name = "default-name"
+snapshot = false
+verbose = false
+very_verbose = false
+
+[peer]
+addr = "127.0.0.1:7001"
+bind_addr = "127.0.0.1:7001"
+ca_file = ""
+cert_file = ""
+key_file = ""
+
+[cluster]
+active_size = 9
+remove_delay = 1800.0
+sync_interval = 5.0
